### PR TITLE
refactor: move flake8 config

### PR DESCRIPTION
**Pull request by dhruvmanila**
### Describe your change:

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?
* [x] Refactoring

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Python files are placed inside an existing directory.
* [ ] All filenames are in all lowercase characters with no spaces or dashes.
* [ ] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Review by cclauss — APPROVED** [review]

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -36,10 +36,6 @@ repos:
     rev: 5.0.4
     hooks:
       - id: flake8
-        args:
```
Comment (cclauss) [inline .pre-commit-config.yaml:39]: Please add a comment on where to find the config settings.

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -51,7 +47,7 @@ repos:
       - id: mypy
         args:
           - --ignore-missing-imports
-          - --install-types  # See mirrors-mypy README.md
+          - --install-types # See mirrors-mypy README.md
```
Comment (cclauss) [inline .pre-commit-config.yaml:50]: I like two space much more. It visually separates the code from the comments

Comment (dhruvmanila) [reply .pre-commit-config.yaml:50]: That's done by the formatter: https://github.com/redhat-developer/yaml-language-server

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -36,10 +36,6 @@ repos:
     rev: 5.0.4
     hooks:
       - id: flake8
```
Comment (cclauss) [inline .pre-commit-config.yaml:38]:
```suggestion
      - id: flake8  # See .flake8 for args
```

**Pull request merged**